e2e/notes.txt

```diff
@@ -0,0 +1,165 @@
+"""Barret Facade classes for working with Shiny inputs/outputs in Playwright"""
```

Comment:
Move relevant info to README and delete this file.

Also, add note to README explaining why some input tests have their own directory.